app/controllers/fees_controller.rb

```diff
@@ -0,0 +1,65 @@
+# frozen_string_literal: true
+
+class FeesController < ApplicationController
+  before_action :set_fee, only: %i[update destroy]
+
+  # GET /fees
+  def index
+    json_response(Fee.all)
+  end
+
+  # POST /fees
+  def create
+    puts create_params
+    fee = Fee.create!(create_params)
```

Comment:
should this be @fee? Im not an expert on ruby syntax so plz disregard if not the case

Reply:
@fee is the same thing as this.fee in Java, while fee without the @ is creating a local variable. In this case, it doesn't matter since nobody accesses this.fee after the call of create — so either would work

Reply:
guys stop tagging this guy Joe 😂

Reply:
LOL

Reply:
Sorry Joe! Resolving this comment 🙃